lmdeploy/serve/turbomind/chatbot.py

```diff
@@ -621,6 +621,7 @@ def stream_consumer(postprocess, res_queue, session, n_input_token,
         """
         status, res, n_token = None, '', 0
         output_ids = np.zeros((1, 1, 0), dtype=np.uint32)
+        text = ''
```

Comment:
If postprocessing failed at the beginning, `n_token` is expected to be equal to `output_ids.shape[-1]`.
https://github.com/InternLM/lmdeploy/blob/e7b2833441be6a4ac374ec191001ba6ae97647e0/lmdeploy/serve/turbomind/chatbot.py#L681-L683

Reply:
The n_token is 0 at first and will be changed after the postprocessing call. But the output_ids.shape[-1] is not 0 because it's the result returned by the model infer call. So if postprocessing failed, the n_token != output_ids.shape[-1].